pkg/security/controller/namespace_scc_allocation_controller.go

```diff
@@ -239,7 +239,13 @@ func (c *NamespaceSCCAllocationController) AddNextPeriodicRepair() {
 	// For this controller to work correctly we need to ensure a periodic repair
 	// of all the range allocations. For that we are adding an artificial key
 	// which will trigger that every 8 hours.
-	delta := time.Duration(8 * time.Hour)
+
+	// ****************************************************************************
```

Comment:
This is explicitly in a separate commit and I have https://bugzilla.redhat.com/show_bug.cgi?id=1856687 to revert that change before shipping 4.6.